### PR TITLE
Add rubocop v0.85 support

### PR DESCRIPTION
**Pull request by toshimaru**

### CODE REVIEW

File: Appraisals [hunk 1]
```diff
@@ -14,3 +14,8 @@ appraise "rubocop-0.84" do
   gem "rubocop", "~> 0.84.0"
   gem "rails"
 end
+
+appraise "rubocop-0.85" do
+  gem "rubocop", "~> 0.84.0"
```
Comment (koic) [inline Appraisals:19]: `0.85.0`?

```suggestion
  gem "rubocop", "~> 0.85.0"
```

File: gemfiles/rubocop_0.85.gemfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+# This file was generated by Appraisal
+
+source "https://rubygems.org"
+
+gem "rubocop", "~> 0.84.0"
```
Comment (koic) [inline gemfiles/rubocop_0.85.gemfile:7]: Ditto.

```suggestion
gem "rubocop", "~> 0.85.0"
```

**Comment by toshimaru** [comment]
Ooops, thanks koic.

**Comment by koic** [comment]
You're welcome. Thanks too :-)

**Comment by toshimaru** [comment]
was lazy copy and past work. ready for review now.

**Pull request merged**